Index: urls.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/urls.py b/urls.py
new file mode 100644
--- /dev/null	
+++ b/urls.py	
@@ -0,0 +1,7 @@
+class Urls:
+    MAIN_PAGE = 'https://qa-scooter.praktikum-services.ru/'
+    ORDER_PAGE = 'https://qa-scooter.praktikum-services.ru/order'
+    ORDER_STATUS_PAGE = 'https://qa-scooter.praktikum-services.ru/track'
+    DZEN_HOME_PAGE = 'dzen.ru'
+    YANDEX_HOME_PAGE = 'ya.ru'
+
